indico/modules/events/contributions/forms.py

```diff
@@ -57,3 +57,16 @@ class ContributionProtectionForm(IndicoForm):
     def __init__(self, *args, **kwargs):
         self.contrib = kwargs.pop('contrib')
         super(ContributionProtectionForm, self).__init__(*args, **kwargs)
+
+
+class SubContributionForm(IndicoForm):
+    title = StringField(_('Title'), [DataRequired()])
+    description = TextAreaField(_('Description'))
+    duration = TimeDeltaField(_('Duration'), [DataRequired()], default=timedelta(minutes=20),
+                              units=('minutes', 'hours'), description=_('The duration of the subcontribution'))
+    speakers = SubContributionPersonListField(_('Speakers'), default_is_submitter=False,
```

Comment:
`default_is_submitter=False`. Is that so? I would say they should have submission rights by default.